service/src/main/kotlin/no/nav/su/se/bakover/service/grunnlag/GrunnlagService.kt

```diff
@@ -28,9 +32,39 @@ internal class GrunnlagServiceImpl(
     }
 
     override fun lagreBosituasjongrunnlag(behandlingId: UUID, bosituasjongrunnlag: List<Grunnlag.Bosituasjon>) {
+        // Se RevurderingServiceImpl og SøknadsbehandlingServiceImpl for opprydning av Behandlingsinformasjon.formue
+        fjernEpsFradragHvisEpsHarEndretSeg(behandlingId, bosituasjongrunnlag)
         return grunnlagRepo.lagreBosituasjongrunnlag(behandlingId, bosituasjongrunnlag)
     }
 
+    private fun fjernEpsFradragHvisEpsHarEndretSeg(
```

Comment:
Tenker at det er sunt at vi får litt større eierskap til konsistens i dataene, har selv tenkt at jeg ønsker meg en "komponent" som kan sjekke konsistens på tvers av grunnlagsdata hver gang vi legger til nytt grunnlag. Koden her må vel kunne sees på som en slags variant av noe sånt. Videre burde vi kanskje tenke litt/ta en runde med design på hvordan vi ønsker at noe slikt skal fungere? Denne varianten tilhører vel det litt mer "invasive" slaget hvor vi bare trår til og fjerner ting - en annen variant kan kanskje være at vi basert på utfallet av "konsistens-komponent" populerer "InformasjonSomRevurderes" med "forslag" til hva saksbehandler må se på. Etterhvert som vi slenger på flere bosituasjoner kan det vel sikkert oppstå situasjoner hvor vi faktisk skal ha både fradrag for EPS i tillegg til at vi ikke har noen EPS. Et spørsmål som kanskje henger litt i lufta (i hvert fall for meg) er om vi bør håndtere denne typen inkonsistenser som "feil" (kommer ikke videre) eller "advarsel" (saksbehandler må se på og 'overstyre').

Reply:
Etter en diskusjon i dag løste vi dette på en annen måte.